cmd/samples/fileprocessing/workflow_test.go

```diff
@@ -31,6 +32,9 @@ func (s *UnitTestSuite) Test_SampleFileProcessingWorkflow() {
 	env := s.NewTestWorkflowEnvironment()
 	env.SetOnActivityStartedListener(func(activityInfo *activity.Info, ctx context.Context, args encoded.Values) {
```

Comment:
Can we change this test to use activity mocks instead?

Reply:
Inside these activities, there're checks for ensuring the activity is running on the same host as other activities. I am not sure how to use activity mocks to do such checks.

Reply:
Makes sense.